data/images.yml

```diff
@@ -0,0 +1,101 @@
+food:
+  - DSC02264
+  - DSC02265
+  - DSC02266
+  - DSC02267
+  - DSC02268
+  - DSC02269
+  - DSC02270
+  - DSC02271
+  - DSC02272
+  - DSC02273
+  - DSC02326
+  - DSC02329
+  - DSC02376
+  - DSC02332
+  - DSC02333
+  - DSC02335
+  - DSC02338
+  - DSC02350
+  - DSC02351
+  - DSC02356
+  - DSC02358
+  - DSC02361
+  - DSC02364
+  - DSC02378
+  - IMG_1486
+  - IMG_1487
+  - IMG_1513
+  - IMG_1516
+  - IMG_1519
+  - IMG_1530
+  - IMG_1533
+  - IMG_1542
+
+challenges:
+  - DSC02317
+  - DSC02320
+  - DSC02321
+
+foodhack:
+  - DSC02301
+  - DSC02305
+  - DSC02311
+  - DSC02313
+  - DSC02314
+  - DSC02331
+  - DSC02339
+  - DSC02340
+  - DSC02341
+  - DSC02365
+  - DSC02398
+  - DSC02402
+  - DSC02430
+  - DSC02444
+  - DSC02517
+  - DSC02522
+  - IMG_1471
+  - IMG_1472
+  - IMG_1473
+  - IMG_1475
+  - IMG_1476
+  - IMG_1477
+  - IMG_1478
+  - IMG_1479
+  - IMG_1481
+  - IMG_1493
+  - IMG_1496
+  - IMG_1503
+  - IMG_1506
+  - IMG_1523
+  - IMG_1551
+
+presentations:
+  - IMG_1554
+  - IMG_1555
+  - IMG_1556
+  - IMG_1557
+  - IMG_1559
+
+teams:
+  - DSC02323
+  - DSC02324
+  - IMG_1489
+  - IMG_1490
+  - IMG_1492
+  - IMG_1509
+  - IMG_1511
+  - IMG_1524
+  - IMG_1535
+  - IMG_1537
+  - IMG_1541
+  - IMG_1548
+  - IMG_1550
+  - IMG_1552
+
+wirsing:
+  - DSC02309
+  - DSC02312
+  - DSC02325
+  - DSC02360
+  - IMG_1518
```

Comment:
Would have probably been less effort to put the images into separate dirs and iterate over them. Makes it also less error-prone when you change something.